### ISSUE TITLE
openssl

### ISSUE THREAD

**Issue by whoizit** [mseri/scrutch#2]
can't build it on updated ArchLinux
```
error: failed to run custom build command for `openssl v0.9.24`
process didn't exit successfully: `/home/user/build/cargo-ssearch/target/release/build/openssl-4e050bcc8f5aae18/build-script-build` (exit code: 101)
--- stderr
thread 'main' panicked at 'Unable to detect OpenSSL version', /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/openssl-0.9.24/build.rs:16:14
note: Run with `RUST_BACKTRACE=1` for a backtrace.

warning: build failed, waiting for other jobs to finish...
error: build failed
```

I have installed this openssl versions:
```
local/openssl 1.1.1-1
local/openssl-1.0 1.0.2.p-1
```

Also I trying install openssl098 and openssl098-dev pkg, but not helps

works with `reqwest         = "0.9.2"`

**Comment by mseri** [comment]
Thanks for the report. I’ll update the code to use the latest reqwest and release a new version by monday

**Comment by mseri** [comment]
Should be fixed by https://github.com/mseri/cargo-ssearch/commit/701b43de45582a42cb1215d2b400e2001ddcb2c2 now released as v0.1.3
